### PR TITLE
if then else expressions

### PR DESCRIPTION
**Pull request by cjllanwarne**
Allows expressions of the form:
```
Boolean b = ...

Int memory = if b then "2GB" else "5GB"
String docker = if b then "ubuntu:latest" else "my:docker"
```

### CODE REVIEW

File: grammar.hgr [hunk 1]
```diff
@@ -49,6 +51,8 @@ grammar {
       javascript: "'([^\\\\\\'\\n]|\\\\[\\\"\\'nrbtfav\\\\?]|\\\\[0-7]{1,3}|\\\\x[0-9a-fA-F]+|\\\\[uU]([0-9a-fA-F]{4})([0-9a-fA-F]{4})?)*'"
     } -> wdl_unescape(:string)
 
+    r'else' -> :else
+    r'then' -> :then
```
Comment (cjllanwarne) [inline grammar.hgr]: Actually suspect this wouldn't be necessary... but if it is, then it's probably also required in the output expression section too

**Comment by geoffjentry** [comment]
don't forget to update the spec

**Comment by katevoss** [comment]
And the changelog :)

**Comment by katevoss** [comment]
@knoblett and @vdauwera there are updates to the Spec, looks good to me but it would be great to get your review too.

**Comment by Horneth** [comment]
👍

**Comment by vdauwera** [comment]
👍

**Pull request merged**